Microsoft.Toolkit.Uwp.SampleApp/SamplePages/RadialGradientBrush/RadialGradientBrushXaml.bind

```diff
@@ -0,0 +1,31 @@
+<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:brushes="using:Microsoft.Toolkit.Uwp.UI.Brushes"
+    mc:Ignorable="d">
+
+  <Grid Background="Purple">
+    <Image Source="/Assets/Photos/NovemberHikeWaterfall.jpg"/>
+    <Rectangle Width="300" Height="300" Stroke="Black" StrokeThickness="2">
+      <Rectangle.Fill>
+        <brushes:RadialGradientBrush 
+                AlphaMode="@[Alpha Mode:Enum:AlphaMode.Premultiplied]"
+                Center="0.5,0.5"
+                ColorInterpolationMode="@[Color Interpolation Mode:Enum:ColorInterpolationMode.SRgbLinearInterpolation]"
+                GradientOrigin="0.5,0.5"
+                Opacity="@[Opacity:DoubleSlider:1.0:0.0-1.0]"
+                RadiusX="@[Radius X:DoubleSlider:0.5:0.0-1.0]"
+                RadiusY="@[Radius Y:DoubleSlider:0.5:0.0-1.0]"
+                SpreadMethod="@[Spread Method:Enum:GradientSpreadMethod.Pad]">
+            <GradientStop Color="Red" Offset="0" />
```

Comment:
Add the colors of the stops to the interactive examples

Reply:
Can't we don't support dynamically binding/animating the stops (add/remove, color, or offset) yet. :(

I spent yesterday looking into it, the callback for the property changed event wasn't firing for some reason.

Reply:
That's because our color dropdown only returns SolidColorBrush via the SolidColorBrushConverter. Should be updated to return Color

Reply:
Doesn't need to be part of this PR

Reply:
I was adding that in for having a Color type for the properties dropdown too.  It was indeed an issue with the GradientStop RegisterPropertyChangedCallback not firing.